cypress/support/commands.ts

```diff
@@ -260,6 +260,26 @@ Cypress.Commands.add('updateChapter', (chapterId, data) => {
   };
   return cy.authedRequest(gqlOptions(chapterMutation));
 });
+/**
+ * Update event using GQL mutation
+ * @param eventId Id of the event
+ * @param data Data of the event. Equivalent of CreateEventInputs for the Events resolver.
+ */
+Cypress.Commands.add('updateEvent', (eventId, data) => {
```

Comment:
Could you add this to the commands declaration (at the bottom of this file)? That way Cypress will understand this command better.

`deleteEvent` is a good example of how to do this.